rct229/rulesets/ashrae9012019/ruleset_functions/get_zone_peak_internal_load_floor_area_dict.py

```diff
@@ -0,0 +1,89 @@
+from rct229.rulesets.ashrae9012019.ruleset_functions.baseline_systems.baseline_system_util import (
+    find_exactly_one_zone,
+    find_exactly_one_schedule,
+)
+from rct229.utils.assertions import getattr_
+from rct229.utils.jsonpath_utils import find_all
+from rct229.utils.pint_utils import ZERO
+
+
+def get_zone_peak_internal_load_floor_area_dict(rmi, zone_id):
+    """
+    Finds the peak coincident internal loads of a zone and returns the value in btu/h/ft2
```

Comment:
It actually doesn't return the result in `btu/h/ft2`.  How about adding "or similar units" after that.